ProjectSidewalk/SidewalkWebpage
# Resolves #1396: add mission screens to validation interface

Resolves #1396.

**Changes**
* Added MVP start mission screen that describes the upcoming mission.
* Added MVP end mission screen that displays how many agree/disagree/not sure labels the user marked.

![image](https://user-images.githubusercontent.com/25534091/53297429-9b9e9800-37d3-11e9-9fa3-844a7d043714.png)

![image](https://user-images.githubusercontent.com/25534091/53297599-65faae80-37d5-11e9-90b5-9e9a6a6900e2.png)


NOTE: Start evaluating this after #1467 has been merged, because the code on this branch relies on some of changes from that branch.

**Testing instructions**
* Start a new mission. You should get the start mission screen. Check that the label id that is displayed matches with the type of labels that you receive during that mission.
* Complete a mission. Check that the number of agree/disagree/not sure labels match the counts that are provided.
* Start another mission. Close your window while it's partially complete, then reopen it so that you get your incomplete mission. Complete the mission and check that the "mission complete" screen comes up. (Note that the counts for agree/disagree/not sure are going to be how many agree/disagree/not sure labels you marked since you began the mission again).
